### DISCUSSION TITLE
OAI_CONFIG_LIST environment variable error with config list

### DISCUSSION THREAD

**Discussion by nhtkid** [microsoft/autogen#468] [Q&A]
Hi,

Many of the notebook example had the following configuration:
`
config_list_gpt4 = autogen.config_list_from_json(
    "OAI_CONFIG_LIST",
    filter_dict={
        "model": ["gpt-4-32k", "gpt-4-32k-0314", "gpt-4-32k-v0314"],
    },
)
`

Isn't that suppose to be:
`
config_list_gpt4 = autogen.config_list_from_json(
    env_or_file="OAI_CONFIG_LIST",
    filter_dict={
        "model": ["gpt-4-32k", "gpt-4-32k-0314", "gpt-4-32k-v0314"],
    },
)
`

As someone with no coding experience, I have spent lots of time wondering why my env wouldn't be recognized. Finally I stumbled to [the last notebook](https://github.com/microsoft/autogen/blob/main/notebook/oai_openai_utils.ipynb) and discovered the missing code. 

Once I used `env_or_file="OAI_CONFIG_LIST"`, my env started working in both Colab and VSCode.

Is I am missing something here, please let me know.
Thanks all.

**Comment by rickyloynd-microsoft** [comment]
Hi @nhtkid. The name of the first parameter to `config_list_from_json` is `env_or_file`, so as long as your code passes "OAI_CONFIG_LIST" as the first parameter in the list, it shouldn't matter whether you include the name `env_or_file=`. Personally, I like the clarity of including the name, but that makes the call longer. Hope that helps.
